webapp/channels/src/components/advanced_create_post/priority_labels.tsx

```diff
@@ -97,6 +94,21 @@ function PriorityLabels({
     priority,
     requestedAck,
 }: Props) {
+    const messages = defineMessages({
+        removePriorityTooltipTitle: {
+            id: 'post_priority.remove',
+            defaultMessage: 'Remove {priority}',
+            values: {priority},
```

Comment:
Have you tested this? I think this doesn't work like this (values go in a separate field when using `intl.formatMessage`) (unless we are using `FormattedMessage` underneath, I don't remember).

Probably as was recommended in a previous comment, using `intl.formatMessage` will make more sense in this context.